src/nano_id_cc/core.cljs

```diff
@@ -49,17 +50,23 @@
     (str (format-time time))))
 
 
+(defn escape
```

Comment:
Because we don't care about double quotes, it can be simplified. So the final function could be as follow: 
```clojure
(defn escape [s] (string/escape s {\' "\\'"}))
```